### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by victorb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
   },
   "dependencies": {
     "async": "^2.1.4",
-    "ipfs": "^0.20.3"
+    "ipfs": "0.20.3",
+    "ipld": "^0.6.0",
```
Comment (daviddias) [inline package.json]: probably you want `ipld-dag-cbor`

Comment (victorb) [reply package.json]: @diasdavid ah, probably! Probably we should update the readme :) https://github.com/ipld/js-ipld-dag-cbor#use-in-a-browser-with-browserify-webpack-or-any-other-bundler

Comment (daviddias) [reply package.json]: done https://github.com/ipld/js-ipld-dag-cbor/commit/108e81afa3191719fd0a125278ab31cf9bb76c4c :) good catch!

Comment (victorb) [reply package.json]: @diasdavid removed it since it's not used (yet), at least in this branch

**Pull request merged**